databuilder/example/scripts/sample_snowflake_table_usage.scala

```diff
@@ -0,0 +1,132 @@
+"""
+This is a example script that parses table names from Snowflake's query logs and calculates their popularity based on
+how many times that same user queries that specific table. You can run this in a Databricks notebook as a job within
```

Comment:
If this can be executed against any snowflake account, can the references to databricks be removed?